src/status_im/contexts/wallet/send/events.cljs

```diff
@@ -294,3 +295,12 @@
                                                 {:event  :wallet/send-transaction
                                                  :error  error
                                                  :params request-params}))}]})))
+
+(rf/reg-event-fx
+ :wallet/wizard.send-forward
```

Comment:
hmm perhaps a name such as
`wallet/send-flow-navigate-forward` will be more descriptive to other devs reading it.

Reply:
Sure 👍

Reply:
Done